modules/integration/src/test/scala/scala/cli/integration/RunScriptTestDefinitions.scala

```diff
@@ -549,4 +551,65 @@ trait RunScriptTestDefinitions { _: RunTestDefinitions =>
       expect(p.out.trim() == "42")
     }
   }
+
+  test("verify os.Path attributes") {
+    // requires os-lib 0.9.2 or later to succeed in Windows
+    val dr      = os.Path.driveRoot
+    val testStr = "/omg"
+    val p       = os.Path(testStr) // <<< must not throw Exception
+    val absPath = p.toString.replace('\\', '/')
+    val relPath = if (dr.nonEmpty) absPath.drop(2) else absPath
+    val synPath = s"${os.Path.driveRoot}$relPath"
+    printf("absPath[%s]\n", absPath)
+    printf("synPath[%s]\n", synPath)
+    printf("relPath[%s]\n", relPath)
+    expect(absPath == synPath)
+    expect(relPath == testStr)
+    expect(absPath endsWith testStr)
+  }
```

Comment:
as mentioned earlier, I don't think it's truly necessary to have this test at all, it's something that should be tested by `os-lib` itself, and not us.

However, if you truly think it's useful for us to catch problems with this (i.e. after the next `os-lib` version bump), then put it together with the other unit tests. It doesn't run Scala CLI, so it doesn't belong in the `integration` directory.

For example, you can add a new test source file in the `cli.tests` file (that's in `modules/cli/src/test/scala/cli/tests`), smth like `WindowsTests`.

However, I think it's also fine to skip on testing this altogether and remove this.

Reply:
removed it